extensions/wikia/ArticleVideo/scripts/featured-video.jwplayer.instant.js

```diff
@@ -78,12 +80,14 @@ require([
 			}
 
 		});
-		console.info('jwplayer after setup');
+		logger.info('jwplayer after setup');
 
 		featuredVideoAds(playerInstance, bidParams);
 		featuredVideoEvents(playerInstance, willAutoplay);
 		featuredVideoTracking(playerInstance, willAutoplay);
 		featuredVideoMoatTracking(playerInstance);
+		logger.subscribeToPlayerSetupError(playerInstance);
+		logger.subscribeToInternalPlayerErrors(playerInstance);
```

Comment:
i'd move this to line 84 before any of submodules is called.
\+ maybe combine them in `subscribeToPlayerErrors`?